package.json

```diff
@@ -23,8 +23,8 @@
   },
   "peerDependencies": {
     "prop-types": "^15.5.4",
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3"
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
```

Comment:
I use the [context api](https://github.com/facebook/react/blob/master/CHANGELOG.md#1660-october-23-2018) to pass along the theme properties. The required version should be **^16.6.0** for now. I'll work on removing the context api in favor of props to allow older versions of react to use this package.